x/datapool/types/keys.go

```diff
@@ -26,9 +26,6 @@ const (
 )
 
 var (
-	// KeyPrefixOracles defines key to store oracle
-	KeyPrefixOracles = []byte{0x01}
```

Comment:
We can reduce all the keys by 1, but it's optional

Reply:
Does it mean reduce all the keys by 1 in `x/datapool` module?

Reply:
yes right!